projects.yaml

```diff
@@ -623,7 +623,7 @@ projects:
   mkdocs_plugin: macros
   github_id: fralau/mkdocs_macros_plugin
   pypi_id: mkdocs-macros-plugin
-  labels: [plugin]
+  labels: [plugin, git-info]
```

Comment:
Labels are not free text, we have a few specific ones, so could you remove the one you added?